play/src/i18n/ca-ES/menu.ts

```diff
@@ -56,6 +56,8 @@ const menu: DeepPartial<Translation["menu"]> = {
         notifications: "Notificacions",
         cowebsiteTrigger: "Preguntar sempre abans d'obrir llocs web i habitacions Jitsi Meet",
         ignoreFollowRequest: "Ignorar peticions de seguir altres usuaris",
+        blockAudio: "Block ambient sounds and music",
+        disableAnimations: "Disable map tile animations",
```

Comment:
```suggestion
```

Best thing is to leave translations empty if not available. The translation library will fallback to English anyway!

Reply:
Done.